### PR TITLE
Add Bucket Web (Actions) documentation

### PR DESCRIPTION
**Pull request by Munter**
### Proposed changes

Add documentation for the new Bucket Web (Actions) destination.
I also made some minor updates to the previous Bucket destination, so the two can refer to each other as alternatives.

### Merge timing

ASAP, once approved, to move the Bucket Web (Actions) closer to public release

### Notes

I was unable to fetch catalog information for Bucket Web (Actions), so I have been unable to verify a correct rendering of the destination info and `components/actions-fields.html` includes. Please advise if I am supposed to do any more on my end in order to get the correct metadata into the collection

localhost_4000_docs_connections_destinations_catalog_bucket-web_

### CODE REVIEW

**Comment by Munter** [comment]
I'm moving this to PR, since the content is done, and I need some feedback on the missing catalog metadata

File: src/connections/destinations/catalog/bucket-web/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    link: '/docs/connections/destinations/catalog/bucket'
+---
+
+{% include content/plan-grid.md name="actions" %}
+
+[Bucket](https://bucket.co/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="blank"} is feature-focused analytics. Bucket empowers software teams with a repeatable approach to shipping features that satisfy customers.
+
+This destination is maintained by Bucket. For any issues with the destination, [contact the Bucket Support team](mailto:support@bucket.co).
+
+{% include content/ajs-upgrade.md %}
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/bucket-web/index.md:18]:
```suggestion
```

This now is automatically accounted for on Segment side so can remove

File: src/connections/destinations/catalog/bucket-web/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+> warning ""
+> If you are using both the "Bucket Web (Actions)"-destination and the server side ["Bucket"-destination](/docs/connections/destinations/catalog/bucket/) on the same source, avoid duplicate event tracking by disabling the "Track Event"-mapping in "Bucket Web (Actions)".
+
+
+## Benefits of Bucket Web (Actions) vs Bucket Classic
+
+Bucket Web (Actions) provides the following benefits over the classic Bucket destination:
+
+- **Clearer mapping of data**. Actions-based destinations enable you to define the mapping between the data Segment receives from your source, and the data Segment sends to the destination.
+- Automatically enables [Live Satisfaction](https://bucket.co/live-satisfaction) prompts in your app, giving you fully automated customer satisfaction scores and feedback on your features.
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/bucket-web/index.md:29]:
```suggestion
- Automatically enables [Live Satisfaction](https://bucket.co/live-satisfaction){:target="_blank"} prompts in your app, giving you fully automated customer satisfaction scores and feedback on your features.
```

**Review by tcgilbert — CHANGES_REQUESTED** [review]
Some minor tweaks, otherwise looks good!

**Comment by tcgilbert** [comment]
@Munter 

> I was unable to fetch catalog information for Bucket Web (Actions)

The integration needs to be live in the catalog for this to populate. so no need to worry about this for now

**Comment by Munter** [comment]
@tcgilbert Thanks for the review! I've added your suggestions.

> The integration needs to be live in the catalog for this to populate. so no need to worry about this for now

Great :)
We've submitted the catalog info for review as well, so I'm crossing my fingers that will be there shortly as well

**Review by tcgilbert — APPROVED** [review]

**Comment by tcgilbert** [comment]
@Munter excellent, I'll have the catalog review done by EOD. And we'll just need to wait for someone from our docs to merge this PR

**Comment by pwseg** [comment]
@Munter I created #5802 to deploy these docs.

**Comment by Munter** [comment]
@tcgilbert @pwseg Thank you both for the swift handling :)

**Pull request closed without merging**